src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -27,24 +27,29 @@ class ApplicationMenu extends Component<Props> {
       showImportExistingProjectPrompt,
     } = this.props;
 
+    const __DARWIN__ = process.platform === 'darwin';
     const template = [
       {
-        label: 'File',
+        label: __DARWIN__ ? 'File' : '&File',
         submenu: [
           {
-            label: 'Create New Project',
+            label: __DARWIN__
+              ? 'Create New Project...'
+              : 'Create &new project...',
             click: createNewProjectStart,
             accelerator: process.platform === 'darwin' ? 'Cmd+N' : 'Ctrl+N',
           },
           {
-            label: 'Import Existing Project',
+            label: __DARWIN__
+              ? 'Import Existing Project...'
+              : '&Import existing project...',
             click: showImportExistingProjectPrompt,
             accelerator: process.platform === 'darwin' ? 'Cmd+I' : 'Ctrl+I',
```

Comment:
Nit - I think line 40 & 47 should also use the `__DARWIN__` constant.

Reply:
@mathieudutour actually already fixes this in #127, I figured there was no reason to add it here since we can just merge it with that PR.

Reply:
ah, OK, you're right. I missed that.